instrumentation/servlet/servlet-3.1/src/main/java/io/opentelemetry/instrumentation/hypertrace/servlet/v3_1/Servlet31Advice.java

```diff
@@ -62,7 +63,7 @@ public static Object start(
 
     HttpServletRequest httpRequest = (HttpServletRequest) request;
     HttpServletResponse httpResponse = (HttpServletResponse) response;
-    Span currentSpan = InstrumentationName.TRACER.getCurrentSpan();
+    Span currentSpan = Servlet3HttpServerTracer.getCurrentServerSpan();
```

Comment:
Using this instead of `Span.current()` bc of this issues https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1661